.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Tests
+
+on:
+  push:
+    branches: ["develop", "release-*"]
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
```

Comment:
May be worth throwing a `"3.x"` in there to automatically pick up the latest stable Python. Up to you.

When I was first learning GHA for unpaddedbase64 I went with just `["3.6", "3.x"]` to cover oldest + newest supported Pythons.

Reply:
Changed it to `["3.6", "3.x"]`, which should be enough, thanks for the suggestion.